### PR TITLE
Create Cart Model make APIs Vendor can create store and add Products …

### PR DESCRIPTION
**Pull request by harmeshverma01**
…Debug in APIs

### CODE REVIEW

File: cart/views.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Create your views here.
+
+
+class CartView(APIView):
+    serializer_class = CartSerializer
+    permission_classes = [permissions.IsAuthenticated]
+    authentication_classes = [authentication.TokenAuthentication]
+    
+    def get(self, request, id=None):
+        cart = Cart.objects.filter(user=request.user)
```
Comment (pythexcel) [inline cart/views.py:16]: Cart can't be more than one for a particular user.

File: order/views.py [hunk 2]
```diff
@@ -11,8 +11,8 @@ class OrderView(APIView):
     serializer_class = OrderSerializer
     
     def get(self, request, id=None):
-        order = Order.objects.get(id=id)
-        serializer = self.serializer_class(order)
+        order = Order.objects.all()
```
[the remark this answers is not in the document]
Comment (harmeshverma01) [reply order/views.py:14]: fixed

File: order/views.py [hunk 3]
```diff
@@ -11,8 +11,8 @@ class OrderView(APIView):
...
+        serializer = self.serializer_class(order, many=True)
         return Response(serializer.data)
     
     def post(self, request):
```
[the remark this answers is not in the document]
Comment (harmeshverma01) [reply order/views.py:18]: fixed

File: order/views.py [hunk 4]
```diff
@@ -32,8 +32,8 @@ class OrderDetailsView(APIView):
     serializer_class = OrderdetailsSerializer
     
     def get(self, request, id= None):
-        order = OrderDetails.objects.get(id=id)
-        serializer = self.serializer_class(order)
+        order = OrderDetails.objects.all()
```
[the remark this answers is not in the document]
Comment (harmeshverma01) [reply order/views.py:35]: fixed

File: cart/views.py (hunk 1 above)
Comment (harmeshverma01) [reply cart/views.py:16]: fixed

**Pull request merged**